model_draft_transactions.go

```diff
@@ -300,7 +299,7 @@ func (m *DraftTransaction) createTransactionHex(ctx context.Context) (err error)
 
 		reserveSatoshis := satoshisNeeded + m.estimateFee(m.Configuration.FeeUnit, 0)
 		if reserveSatoshis <= dustLimit && !m.containsOpReturn() {
-			m.client.Logger().Error(ctx, "amount of satoshis to send less than the dust limit")
+			m.client.Logger().Error().Msg("amount of satoshis to send less than the dust limit")
```

Comment:
Do we have any information that would help us find this log? ID or smth?

Reply:
Added txID